iop/usb/usbmass_bd/src/usb_mass.c

```diff
@@ -15,7 +15,7 @@
 #include <usbhdfsd-common.h>
 
 // #define ASYNC
-// #define DEBUG  //comment out this line when not debugging
```

Comment:
This line I suppose need to keep commeted

Reply:
Yes you’re right, this pr isn’t “complete” but more so just to get feedback on the overall changes